src/antsibull_docs/cli/doc_commands/stable.py

```diff
@@ -166,7 +169,8 @@ def normalize_plugin_info(plugin_type: str,
             # But we can use the default value (some variant of "empty") for everything else
             # Note: We looped through doc first and returned an exception if doc did not normalize
             # so we're able to use it in the error message here.
-            errors.append(f'Unable to normalize {new_info["doc"]["name"]}: {field}'
+            plugin_name = (new_info["doc"] or {}).get("name") or "???"
```

Comment:
I would appreciate a comment in the code explaining how/why the string `"???"` could end up here

Reply:
I added a comment in 3b0a9328290d1138d386d09feb3ae78601d7beee, and at the same time decided to rewrite that function a bit.

Reply:
Actually I undid most changes to this function.